lib/pool.dart

```diff
@@ -125,6 +125,97 @@ class Pool {
     }
   }
 
+  /// Returns a [Stream] containing the result of [action] applied to each
```

Comment:
Please document ordering behavior. Do elements emit on the stream in the order that their `action` completes, or in order that the input appeared in `sourceItems`?

Reply:
done